benchmarks/libxml2-v2.9.2/build.sh

```diff
@@ -44,5 +36,5 @@ cd -
 
 build_lib
 
-$CXX $CXXFLAGS -std=c++11 $SCRIPT_DIR/target.cc -I BUILD/include BUILD/.libs/libxml2.a $FUZZER_LIB -o $FUZZ_TARGET
-wget https://raw.githubusercontent.com/google/AFL/debe27037b9444bbf090a0ffbd5d24889bb887ae/dictionaries/xml.dict -O $FUZZ_TARGET.dict
+$CXX $CXXFLAGS -std=c++11 $SRC/target.cc -I BUILD/include BUILD/.libs/libxml2.a $FUZZER_LIB -o $OUT/fuzz-target
+wget https://raw.githubusercontent.com/google/AFL/debe27037b9444bbf090a0ffbd5d24889bb887ae/dictionaries/xml.dict -O $OUT/fuzz-target.dict
```

Comment:
Should this wget be in Dockerfile so that download happens once, same for other places.